src/domain_bridge/domain_bridge.cpp

```diff
@@ -150,8 +145,12 @@ class DomainBridgeImpl
     const std::string & topic,
     const std::string & type,
     size_t from_domain_id,
-    size_t to_domain_id)
+    size_t to_domain_id,
+    const TopicBridgeOptions & options)
   {
+    // TODO(jacobperron): Do something with options.callback_group()
```

Comment:
this can be passed to `create_publisher/create_subscription` through `PublisherOptions/SubscriptionOptions`.

Reply:
That's what I thought, but we aren't calling rclcpp's `create_publisher/create_subscription` and instead using the `rclcpp::PublisherBase` and `rclcpp::SubscriptionBase` directly, which only accepts the `rcl` options struct.

Reply:
Nevermind, I see where I can pass the callback group. It must have been too late in the day yesterday :expressionless:

Reply:
Done a290340